CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## Version 0.1.12: TBD
+
+* Clicking on status bar opens output panel,
+  [#177](https://github.com/clangd/vscode-clangd/pull/177)
+* Commands to open user and project clangd confguration files
+  [#181](https://github.com/clangd/vscode-clangd/pull/181)
+* Inlay hints support for parameters
+  [#168](https://github.com/clangd/vscode-clangd/pull/168) and types
+  [#188](https://github.com/clangd/vscode-clangd/pull/188)
```

Comment:
May want to mention that this only works with latest clangd with experimental `-inlay-hints` option, to set expectations